tmk_core/common/matrix.h

```diff
@@ -31,6 +31,16 @@ typedef  uint32_t   matrix_row_t;
 #error "MATRIX_COLS: invalid value"
 #endif
 
+#if (MATRIX_ROWS <= 8)
```

Comment:
@That-Canadian I think this will fix the issue with the split boards I was having